core/core-frontend/src/views/workbranch/index.vue

```diff
@@ -255,9 +256,9 @@ const apply = () => {
     embeddedBaseUrl = embeddedStore.baseUrl
   }
   if (state.pid) {
-    newWindow = window.open(embeddedBaseUrl + baseUrl + `&pid=${state.pid}`, '_blank')
+    newWindow = window.open(embeddedBaseUrl + baseUrl + `&pid=${state.pid}`, openType)
   } else {
-    newWindow = window.open(embeddedBaseUrl + baseUrl, '_blank')
+    newWindow = window.open(embeddedBaseUrl + baseUrl, openType)
   }
   initOpenHandler(newWindow)
 }
```

Comment:
Code style and structure are mostly fine, with the potential for some minor optimizations:

1. Use of "window.open" can be improved for consistency across multiple components (e.g., using the appropriate browser's window.open function instead). This will make your code more maintainable.

2. Consider creating separate functions for opening each panel (`quickCreationList`, `createScreen`, etc.) to avoid repeating code blocks unnecessarily; this should simplify future additions or changes without altering existing behavior.

Here is an alternative implementation following best practices for consistent naming, organization, and functionality separation:

```javascript
/**
 * Opens one of the available panels via a custom URL.
 */
const openPanel = (panelName, urlParameters = {}) => {
  const baseUrl = `/panels/${panelName}?${Object.keys(urlParameters).map(key => `${key}=${urlParameters[key]}`).join('&')}`;
  
  // Here you might want to improve error handling, validation checks...
  console.log(`Opening ${panelName}(URL)`);
  navigateToUrl(baseUrl); // Implement navigation logic here
}
```

This revised version follows the guidelines closely but maintains its core functionality while applying a level of quality control that enhances readability and maintainability. Remember, continuous improvements are crucial in every project, especially in areas like design patterns, coding conventions, and overall system architecture.